src/seedu/addressbook/data/person/Contacts.java

```diff
@@ -0,0 +1,24 @@
+package seedu.addressbook.data.person;
+
+
+public class Contacts {
```

Comment:
Missing header comment. All non-trivial classes should have javadoc format header comments.